### DISCUSSION TITLE
ja-JP Language(日本語)

### DISCUSSION THREAD

**Discussion by RyoTagami** [ikas-mc/ContextMenuForWindows11#171] [Languages]
I created ja-JP custom language file.
[ja-JP.json](https://github.com/user-attachments/files/17350038/ja-JP.json)

**Comment by nihongo-helper0119** [comment]
The Japanese (ja-JP) custom language file has been updated for v5.8.1.
[ja-JP.json](https://github.com/user-attachments/files/28708924/ja-JP.json)
